Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -40,13 +41,59 @@ public partial class MainWindow : Window
         private ManifestModel config;
         private ModuleModel[]? modules;
 
+        private string commandLineURL;
+        private string[] hereIsMyCode = (Application.Current as App).CustomerCode;
+
         public MainWindow()
         {
             InitializeComponent();
 
-            config = new ManifestParser().Manifest;
-            modules = config.Modules;
-            DataContext = modules;
+            this.Loaded += MainWindow_Loaded;
+        }
+
+        private void MainWindow_Loaded(object sender, RoutedEventArgs e)
+        {
+            if (this.hereIsMyCode.Length != 0)
+            {
+                this.LazyLoading();
+            }
+            else
+            {
+                config = new ManifestParser().Manifest;
+                modules = config.Modules;
+                DataContext = modules;
+            }
+        }
+
+        private void LazyLoading() {
+            Dictionary<string, string> commands = new Dictionary<string, string>();
+
+            this.hereIsMyCode.ToList<string>().ForEach(item => {
+                item = item.TrimStart('-');
+                string[] command = item.Split("=");
+
+                commands.Add(command[0], command[1]);
+            });
```

Comment:
Don't invent the wheel, use the `System.CommandLine` library instead, it's well tested, robust, and pretty standard.